client/src/components/Header/index.spec.js

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+import Header from './index';
+
+describe('Header component', () => {
+    it('should be rendered with default props', () => {
+        const HeaderComponent = shallow(<Header />);
```

Comment:
for me snapshot has the following value `ShallowWrapper {}`
which is not useful at all. Please, use https://www.npmjs.com/package/enzyme-to-json